sources/osgShadow/shaders/shadowsReceiveMain.glsl

```diff
@@ -1,30 +1,65 @@
 
+    // 0 for early out
+    bool earlyOut = false;
+    // Calculate shadow amount
+    float shadow = 1.0;
     if (!lighted)
-        return 0.;
+    {
+        shadow = 0.0;
+        earlyOut = true;
+    }
 
     if (depthRange.x == depthRange.y)
-        return 1.;
+      earlyOut = true;
```

Comment:
indentation is wrong, plz pass the file to a glsl mode in emacs or something similar, also check the how bracket are in other files like https://github.com/cedricpinson/osgjs/blob/9c9b8a7fff32bcc9241ac5db3e6dabecc02d576c/sources/osgShader/node/lights.glsl use the same coding style plz